### ISSUE TITLE
Issue new release of TurtleFSI to generate DOI for Nature Comms Paper

### ISSUE THREAD

**Issue by dbruneau-mie** [KVSlab/turtleFSI#92]
Hi @keiyamamo @jorgensd @johannesring @KristianValen-Sendstad, I would like to issue a new release (v2.2.1) of TurtleFSI to generate a DOI in Zenodo, so I can reference it in my Nature Comms paper. Do you have any concerns about doing this?

**Comment by keiyamamo** [comment]
There is no conners from my side.

**Comment by johannesring** [comment]
No problem, but it would be good to bump the version number when making a release. There have been a few releases lately, but `setup.py` is still at version 1.5.

**Comment by KristianValen-Sendstad** [comment]
No objection from me, DAB.

KVS

**Comment by dbruneau-mie** [comment]
Ok, I will increment the version number in setup.py. Have the recent
versions code been passing the built-in pytest?

DAB

**Comment by johannesring** [comment]
Yes, the tests are passing (see [here](https://github.com/KVSlab/turtleFSI/actions)), but the badge at the top of [README.md](https://github.com/KVSlab/turtleFSI/blob/master/README.md) is pointing to a Travis test that does not seem to be running.

**Comment by keiyamamo** [comment]
@dbruneau-mie  test is automatically run with github everyday. @johannesring We should probably remove travis from readme as well as [.travis.yml](https://github.com/KVSlab/turtleFSI/blob/master/.travis.yml)

**Comment by johannesring** [comment]
@keiyamamo - yes, I will do that.

**Comment by keiyamamo** [comment]
@johannesring Thanks! For some reason, documentation badge says “failing” even though the documentation looks fine. It would be nice to fix it but I’m not sure when the documentation is built in the current setting.

**Comment by johannesring** [comment]
I guess the documentation on readthedocs is autobuilt on commits. It is probably configured on readthedocs.

**Comment by dbruneau-mie** [comment]
I created a new release, but could not get zenodo to automatically create a DOI. I ended up archiving an older version of the repo manually at: 

https://zenodo.org/doi/10.5281/zenodo.10001935
